front/src/styles/scss/applications/stdcm/_card.scss

```
@@ -92,9 +89,11 @@
     }
 
     .feed-back {
-      --field-wrapper-padding-bottom: 9px;
-      padding-top: 3px;
-      padding-bottom: 3px;
+      padding: 0;
+      &.error {
+        --field-wrapper-padding-bottom: 0px;
+        --status-message-wrapper-tooltip--offset: 0px;
+      }
```

To match the mockup (this *** `padding: 0` breaks all osrd-ui styles, I'll fix it in the next iteration)
```suggestion
      &.error {
        --field-wrapper-padding-bottom: 5px;
        .status-message-wrapper-tooltip.tooltip-left {
          left: -4px;
        }
```